File: backend/beverage.go
package pe

import "time"

type Beverage struct {
	ID              string        `json:"beverageId"`
	Name            string        `json:"name"`
	Desc            string        `json:"description"`
	ImageSrc        string        `json:"imageSrc"`
	IsActive        bool          `json:"isActive"`
	LastPriceUpdate time.Time     `json:"lastUpdate"`
	Params          PricingParams `json:"-"`
}

type PricingParams struct {
	MaxPrice      float64
	MinPrice      float64
	BasePrice     float64
	BuyMultiplier float64
	HalfTime      time.Duration
}

File: BeveragePaymentApi/BeveragePaymentApi/Domain/Entities/Beverage.cs
using System.ComponentModel.DataAnnotations;

namespace BeveragePaymentApi.Domain;

public class Beverage
{
    [Key]
    public int BeverageId { get; set; }
    [MaxLength(100)]
    public string? Name { get; set; }

    public string? Description { get; set; }

    public string? ImageSrc { get; set; }

    public double BasePrice { get; set; }

    public double MaxPrice { get; set; }

    public double MinPrice { get; set; }
    public double BuyMultiplier { get; set; }
    public string? HalfTime { get; set; }
    public bool IsActive { get; set; }
    public int TotalSales { get; set; }
    public List<Price>? Prices { get; set; }

}